### ISSUE TITLE
Problems with training

### ISSUE THREAD

**Issue by qingfei-ops** [THU-MIG/yoloe#105]
Can the training set and the validation set in the training file be set differently?
